cmd/yorkie/context/list.go

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2022 The Yorkie Authors. All rights reserved.
```

Comment:
It would be nice to change `2022` to `2023` for all the files that you have created.